### PR TITLE
7. Data model

### PR DESCRIPTION
**Pull request by libdx**
Define core models with admin site support.

### CODE REVIEW

File: myflights/apps/core/models.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django_countries.fields import CountryField
+
+
+class BaseModel(models.Model):
+    """Base abstract model
```
Comment (pioole) [inline myflights/apps/core/models.py]: Please add dot at the end of line.

File: myflights/apps/core/models.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    openflights_id = models.IntegerField(
+        _('ID from OpenFlights database'), null=True, blank=True
+    )
+
+    def __str__(self):
+        return f'{self.name} {self.callsign}'
+
+
+class Route(BaseModel):
+    """Represents routes between airports
```
Comment (pioole) [inline myflights/apps/core/models.py]: Same.

File: myflights/apps/core/models.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    equipment = models.CharField(
+        _('3-letter codes for plane type'), max_length=100, null=True, blank=True
+    )
+
+    def __str__(self):
+        return f'{self.origin_airport} - {self.destination_airport}'
+
+
+class Flight(BaseModel):
+    """Represents single flight along it's route
```
Comment (pioole) [inline myflights/apps/core/models.py]: Same

File: myflights/apps/core/models.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    name = models.CharField(_('Airport name'), max_length=255, null=True)
+    city_name = models.CharField(_('City name'), max_length=255, null=True, blank=True)
+    country = CountryField(_('Country'))
+    iata = models.CharField(
+        _('3-letter IATA code'), max_length=3, null=True, blank=True
+    )
+    icao = models.CharField(
+        _('4-letter ICAO code'), max_length=4, null=True, blank=True
+    )
+    latitude = models.FloatField(_('Latitude'))
```
Comment (pioole) [inline myflights/apps/core/models.py]: Why is this needed?

File: myflights/apps/core/tests.py [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+
+class BaseTestCase(TestCase):
+    def _create_airline(self, name='Abc Ltd.', country='AU'):
+        airline = Airline.objects.create(
+            name=name,
+            alias='A',
+            iata='xyz',
+            icao='defg',
+            callsign="ABC",
```
Comment (pioole) [inline myflights/apps/core/tests.py]: `"` vs `'`

File: myflights/apps/core/models.py (hunk 4 above)
Comment (libdx) [reply myflights/apps/core/models.py]: Which one, `icao` or `latitude`? Anyways, I've imported most of the columns from archives, complicating model to simply see how to define different fields with different options and mimic more realistic data model (for the sake of training).
